File: practice/codewars/luck_check.js
function luckCheck(ticket) {
  var first = 0;
  var second = 0;
  for (var i = 0; i < ticket.length / 2; i++ ) {
    first += parseInt(ticket[i])
  }
  for (var i = ticket.length / 2; i < ticket.length; i++ ) {
    second += parseInt(ticket[i])
  }
  if (first === second) {
    return true
  }
  return false;
};

File: practice/codewars/luck_check.py
def luck_check(string):
    first = 0
    for n in range(0, int(len(string) / 2 ) ):
        first += int(string[n])
    second = 0
    if len(string) % 2 == 0:
        for n in range(int(len(string) / 2 ),len(string)):
            second += int(string[n])
    else:
        for n in range(int(len(string) / 2 + 1 ),len(string)):
            second += int(string[n])

    if first == second:
        return True
    return False
